qa/tasks/cephfs/test_volumes.py

```diff
@@ -35,15 +35,18 @@ def _fs_cmd(self, *args):
     def _raw_cmd(self, *args):
         return self.get_ceph_cmd_stdout(args)
 
-    def __check_clone_state(self, state, clone, clone_group=None, timo=120):
+    def __check_clone_state(self, states, clone, clone_group=None, timo=120):
+        if isinstance(states, str):
+            states = (states, )
```

Comment:
preferably: `tuple(states)`

Reply:
That won't work, it create a tuple out of characters of the string of initializing a tuple of strings where `states` is only member. 

See -
```
$ python
Python 3.11.9 (main, Apr 17 2024, 00:00:00) [GCC 13.2.1 20240316 (Red Hat 13.2.1-7)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> x = 'abcd'
>>> y = tuple(x)
>>> y
('a', 'b', 'c', 'd')
>>>
>>>
>>>
>>> i = 1
>>> j = tuple(i)
```

Reply:
Marking this resolved.